spec/enumerable_spec.rb

```diff
@@ -0,0 +1,160 @@
+require('./lib/enumerable.rb')
+include Enumerable
+
+class MockObject
+  attr_reader :variable, :other_variable
+  def initialize variable = 'hello', other_variable = 'world'
+    @variable = variable
+    @other_variable = other_variable
+  end
+end
```

Comment:
What's the idea behind this?

Reply:
:) Just mocking and try to test our module can work with objects. But I know that everything is object in ruby :)